ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -51,6 +51,21 @@
     <input type="hidden" name=":formstart" value="${resource.path}"/>
     <input type="hidden" name="_charset_" value="utf-8"/>
     <input data-sly-test="${container.redirectUrl}" type="hidden" name=":redirect" value="${container.redirectUrl}"/>
+    <div data-sly-test="${container.properties['fd:hamburgerMenu']}" class="cmp-adaptiveform-container-hamburger-menu-container">
+        <div class="cmp-adaptiveform-container-menu">
+            <div role="navigation" aria-label="Toggle menu" class="cmp-adaptiveform-container-hamburger-menu-top-bar">
+                <button aria-controls="hamburger-menu" type="button" class="cmp-adaptiveform-container-hamburger-icon" aria-label="Toggle menu"></button>
```

Comment:
The CSS classes are not BEM compliant, menu, menu top bar etc should be part of element and not a block. Please revisit all BEM classes and change them

Reply:
Revisited all BEM classes and changed acc to BEM principles.